appserver/admingui/common/src/main/java/org/glassfish/admingui/common/util/RestUtil.java

```diff
@@ -885,7 +885,7 @@ public static RestResponse post(String address, Object payload, String contentTy
         }
         Response cr = target.request(RESPONSE_TYPE).header("Content-Type", contentType)
                 .cookie(new Cookie(REST_TOKEN_COOKIE, getRestToken()))
-                //                .header("Content-type", MediaType.APPLICATION_FORM_URLENCODED)
+                .header("Content-type", MediaType.APPLICATION_FORM_URLENCODED)
```

Comment:
Danger Will Robinson!
I'm very wary of this change - I need to give this a thorough bashing to ensure it doesn't blow something up.
**Lots** of things use this method, sending all kinds of gubbins through it e.g. Rest Management Interface sending Json

Reply:
I see, I think it can be safe to revert, the jsf specifies the content type when it calls the rest request